### PR TITLE
[iDEAL] Retry failed transaction

### PR DESCRIPTION
**Pull request by mpetrenco**
- Added retry button to the TransactionStatus view;
- Retry button tap handled by IDEALFormViewController;
- Calls `startPoolingStatus` again;

### CODE REVIEW

File: Source/iDEAL/IDEALFormViewController.m [hunk 1]
```diff
@@ -168,6 +169,23 @@ - (void)displaySavedBankIfNeeded {
     [self didSelectBank:storedBank];
 }
 
+# pragma mark - Private methods
+
+- (void)startPoolingStatus {
+    
+    [self.idealManager poolTransactionStatusForOrderId:self.orderId
```
Comment (gcojocaru) [inline Source/iDEAL/IDEALFormViewController.m]: pollTransactionStatusForOrderId

File: Source/iDEAL/IDEALFormViewController.m [hunk 2]
```diff
@@ -168,6 +169,23 @@ - (void)displaySavedBankIfNeeded {
     [self didSelectBank:storedBank];
 }
 
+# pragma mark - Private methods
+
+- (void)startPoolingStatus {
```
Comment (gcojocaru) [inline Source/iDEAL/IDEALFormViewController.m]: startPollingStatus

File: Source/iDEAL/IDEALFormViewController.m (hunk 1 above)
Comment (mpetrenco) [reply Source/iDEAL/IDEALFormViewController.m]: D:

Comment (mpetrenco) [reply Source/iDEAL/IDEALFormViewController.m]: Done.

File: Source/iDEAL/IDEALFormViewController.m (hunk 2 above)
Comment (mpetrenco) [reply Source/iDEAL/IDEALFormViewController.m]: Done.

**Pull request merged**